### ISSUE TITLE
Provide an option for casing SQL Keywords (aduxr)

### ISSUE THREAD

**Issue by gengelbms** [Microsoft/azuredatastudio#4926]
Issue Type: <b>Feature Request</b>

You can often create SQL code that vaguely resembles a ransom note. If your style for writing  SQL keywords does not match what the upper case style used by code completion. Code completion writes SQL keywords in all  upper case (SELECT), but users write these keywords in all lower case (select), title case (Select), or camel case (OpenRowset). 

Let users choose how they want their SQL keywords written.

![image](https://user-images.githubusercontent.com/41927913/55818597-d3278200-5aab-11e9-91d1-b2cae115fac5.png)

Azure Data Studio version: Azure Data Studio 1.5.2 (f74080c96310354bc92211f1826c0fdce78ca2f6, 2019-03-22T06:09:10.602Z)
OS version: Darwin x64 18.5.0

**Comment by kburtram** [comment]
@gengelbms there is an setting named `Mssql > Format: Keyword Casing` that controls this behavior.  Currently we only support UPPERCASE and lowercase but the other options seem reasonable as well.

**Comment by gengelbms** [comment]
Thank you. Most of the participants I've seen seem to prefer lowercase, so having the setting is great. I have seen several camel-case user, so that option would be nice have. I would guess that current settings give us about 90% coverage.

**Comment by kazinad** [comment]
> @gengelbms there is an setting named `Mssql > Format: Keyword Casing` that controls this behavior. Currently we only support UPPERCASE and lowercase but the other options seem reasonable as well.

I changed this setting to lowercase, but the code completion list still provides me only uppercase items even after restarting the application.

**Comment by kburtram** [comment]
@kazinad are you using Postgres extension?  This option is currently only supported for MSSQL.

**Comment by kazinad** [comment]
> @kazinad are you using Postgres extension? This option is currently only supported for MSSQL.

Yes I use MSSQL, but there are 3 options related to casing in settings, and I changed possibly the wrong one. Thx
